lit_gpt/config.py

```diff
@@ -590,9 +590,26 @@ def norm_class(self) -> Type:
         norm_eps=1e-5,
         _mlp_class="LLaMAMLP",
         intermediate_size=28672,
+    ),
+    
+    # https://huggingface.co/stabilityai/stablecode-completion-alpha-3b/blob/main/config.json
```

Comment:
Since this is a diferent model family to FreeWilly (even though both are from StabilityAI), I suggest that a new "section" is added at the bottom.